sunbird_service_api_test/src/test/resources/templates/otp/verify/testVerifyOtpWithInvalidPhoneFailure/request.json

```diff
@@ -0,0 +1,8 @@
+{
+    "request": {
+        "key" : "invalidPhoneNumber",
```

Comment:
use like 10+digits as validation or introduce one more failure case. 

In general we might be checking simply for 10 digits number as validtion in this case.